cfme/markers/env_markers/provider.py

```diff
@@ -196,9 +194,21 @@ class ProviderEnvironmentMarker(EnvironmentMarker):
 
     def process_env_mark(self, metafunc):
         if hasattr(metafunc.function, self.NAME):
-            args = getattr(metafunc.function, self.NAME).args
-            kwargs = getattr(metafunc.function, self.NAME).kwargs.copy()
-
+            mark = None
+            for mark in getattr(metafunc.function, self.NAME):
+                if 'override' in mark.kwargs.keys() and mark.kwargs['override']:
```

Comment:
this should be `kwargs.get('override', False)`

Reply:
It should, I'm not sure why I did that, I _think_ I had had some issues with treating it as a dict or something.....